package.json

```diff
@@ -1,33 +1,34 @@
 {
   "peerDependencies": {
     "react": ">=0.13.0 <16.0.0"
```

Comment:
Want to update the value here to support 16?

Reply:
I will change it